db.h

```diff
@@ -12,7 +12,7 @@
 
 #define MIN_RETRY 1000
 
-#define REQUIRE_VERSION 170013
+#define REQUIRE_VERSION 170014
```

Comment:
We should not bump this yet; it will cause the seeders to immediately stop advertising any pre-170014 peers, which could inhibit integration into the network as nodes restart. We should instead bump this after EoS halt.

Reply:
Good point

Reply:
@str4d Do we only want to deploy this on the testnet seeder still though? Just not mainnet for reasons above, correct?

Reply:
I think the same rationale applies even more strongly for testnet, as there are way fewer peers.

Reply:
So we should bump this after the later of "mainnet EoS halt" and "testnet NU5 activation".

Reply:
Hmm, actually now that I think about it, I think we probably should make this change once we've seen at least 50% of the mainnet and testnet upgrade to 4.5.0.